### PR TITLE
Update help window for updated usage of edit command

### LINKED ISSUE #73: Update help shown to user with new syntax for add/remove tag
With the new delete tag prefix `dt/` for the edit command, the help command/window should show this as a possible syntax as well.

### PR DESCRIPTION
**Pull request by sayomaki**
Resolves #73

### CODE REVIEW

File: src/main/java/seedu/address/ui/HelpWindow.java
Comment (Tkaixiang) [inline src/main/java/seedu/address/ui/HelpWindow.java:27]: You might want to consider remove the `-` since it is part of the same point

**Review by Tkaixiang — APPROVED** [review]
pre-approved pending minor change above ^

File: src/main/java/seedu/address/ui/HelpWindow.java
Comment (Vatinius) [reply src/main/java/seedu/address/ui/HelpWindow.java:27]: On a similar note, i've noticed something similar at line 53 of HelpWindow.java. It results in a weird-looking text format (see screenshot below). Maybe you could look into resolving that too with this PR?
image

If not, I can create another PR just to resolve it.

File: src/main/java/seedu/address/ui/HelpWindow.java
Comment (Vatinius) [inline src/main/java/seedu/address/ui/HelpWindow.java:54]: There is an extra whitespace at the beginning of the sentence! You might want to remove it before merging.

**Comment by Vatinius** [comment]
lgtm!

**Pull request merged**
